README.markdown

```diff
@@ -78,4 +77,4 @@ We Are Working On
 * The aggregation functionality is ignored completely
 * The code is... a mess. Deadlines do that to you, we're working on it
 * The default table schema isn't indexed all the places it needs to be
-* Easier ways to diff URLs
\ No newline at end of file
+* Easier ways to diff URLs
```

Comment:
What was changed here?

Reply:
Oops, no idea! I'm guessing it was just whitespace, I'll try and revert this bit.

Reply:
Dang it, there's an extra new line at the end of the file but it won't let me remove it. I'll just create a new PR and hopefully it won't have the newline this time.

Reply:
You could have edited this one as well.

Reply:
Yeah, I tried editing (twice) and it was still the same, so I thought I'll just try again from scratch. Still no luck as it turned out :) Sorry for the duplicates!